onnx/backend/test/case/node/gelu.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) ONNX Project Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import math
+
+import numpy as np
+
+import onnx
+from onnx.backend.test.case.base import Base
+from onnx.backend.test.case.node import expect
+
+
+class Gelu(Base):
+    @staticmethod
+    def export() -> None:
```

Comment:
I would mention tanh in the method name since it is not the default behaviour.

Reply:
Done. Changed the name to export_gelu_tanh